library/spdm_secured_message_lib/libspdm_secmes_encode_decode.c

```diff
@@ -573,8 +607,13 @@ libspdm_return_t libspdm_decode_secured_message(
 
     default:
         LIBSPDM_ASSERT(false);
+        /*zero salt for security*/
+        libspdm_zero_mem(salt, LIBSPDM_MAX_AEAD_IV_SIZE);
         return LIBSPDM_STATUS_UNSUPPORTED_CAP;
     }
 
+    /*zero salt for security*/
```

Comment:
Please remove `/*zero salt for security*/`. No need to say it every where.

Reply:
Thanks. I have removed the comments except the first comment.